data/bactrian.txt

```diff
@@ -0,0 +1,379 @@
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+mission "Bactrian Unlock [0]"
+	minor
+	name "Locate Missing Scientists"
+	description "A Deep Star Queen transporting top scientists has gone missing. It was last seen near the Betelgeuse system in the Far North. Find its whereabouts and return it to <destination> if possible."
+	source
+		attributes deep
+	destination Valhalla
+	waypoint Betelgeuse
+	to offer
+		has "Smuggler's Den, Part 2: done"
+		has "Expedition to Hope 2: done"
+		has "WR Star 1: done"
+		has "Rescure Miners 2: done"
+		has "Humanitarian 2: done"
+		has "Terraforming 5: done"
+	on offer
+		conversation
+			`While you are watching one of the telescreens in a restaurant located in the spaceport, you are approached by a man and a woman, both wearing Deep Security uniforms.`
+			`	"Hello. Are you <first> <last>?" the man asks`
+			choice
+				`	"Yes I am. Is there something you need?"`
+					goto named
+				`	"Sorry, but I'm not the person you're looking for."`
+				
+			`	The man looks a little embarrassed after you say that. "Alright. Sorry for bothering you." The two make their way out of the restaurant and you return to watching the telescreen.`
+				decline
+	
+			label named
+			`	"We were wondering if you would be able to assist us. We have a record of you having assisted in a number of humanitarian ventures before."`
+			`	The woman pulls out a tablet of some sort and begins reading from it. "Helping to terraform Rand, rescuing miners on a Syndicate world, transporting a few scientists conducting research on Hope." She continues on to mention a few more situations that you were involved in and finally ends on you helping Joe and Maria off of Smuggler's Den, although she doesn't mention them by name.`
+			`	The man begins speaking to you after the woman stops. "Rather impressive work, I might say."`
+			choice
+				`	"Wait, you have a record on me?"`
+				`	"What do you need me to do?"`
+					goto mission
+	
+			`	"The Deep keeps a record on a lot of things, especially when we notice a certain individual doing a lot of humanitarian deeds."`
+			branch plot
+				has "main plot completed"
+			`	The man begins to tell you about the mission.`
+				goto mission
+	
+			label plot
+			`	"Besides," the woman chimes in, "what do you expect after having been involved in the war?" The man begins to tell you about the mission.`
+	
+			label mission
+			`	"We need you to find a Star Queen that recently went missing in the Far North region and return to <destination> once you have located it. Aboard this Star Queen was a number of top scientists intending to do research on the Wolf-Rayet star in Gorvi, which, due to its proximity to the northern anarchist worlds, has very little known about it."`
```

Comment:
Was is singular. Were is plural. Whichever word is coming after Star Queen is refering to the scientists, and since there are multiple scientists, it should be were.